app/controllers/policies_controller.rb

```diff
@@ -0,0 +1,26 @@
+require_relative '../requesters/create_policy_requester'
+
+class PoliciesController < BaseController
+  before do
+    redirect '/login' unless user_signed_in?
```

Comment:
Sugestão: o que acha de padronizar o uso do `unless`, `if not` ou do `!` de negação nesses redirecionamentos de rota?